docs/developer-guide/project-structure.md

```diff
@@ -1,59 +1,118 @@
 # Project structure
-
 ## Modules
-
 ### Core Modules
 
-All the core modules are located under the [Strongbox] project. 
-Each of these modules has a `README.md` file explaining briefly what kind of code contains. 
-New modules should also follow the same format.
+All the core modules are located under the [Strongbox] project. Each of these modules has a `README.md` file explaining briefly what kind of code contains. New modules should also follow the same format.
 
 This is a brief breakdown of the modules:
 
-* [strongbox-client]/  
+* [strongbox-aql](https://github.com/strongbox/strongbox/tree/master/strongbox-aql)
+    This is where the [Artifact Query Language (AQL)](https://strongbox.github.io/user-guide/artifact-query-language.html) is located.
+    
+* [strongbox-client]    
     This is where the artifact client resides.
-* [strongbox-commons]/  
-    This is where the most common code which is across other modules resides.
-* [strongbox-event-api]/    
-    Contains an event API.
-* [strongbox-metadata-core]/  
-    This contains the most commonly needed code related to Maven metadata.
-* [strongbox-parent]  
-    This is the Maven parent which is (and should be) inherited by all modules. It is the right (and only place) 
-    to define versions for dependencies and plugins. Versions of dependencies should not be defined in any other 
-    `pom.xml` files in order to ease the maintenance of these across the codebase.
-* [strongbox-resources]/  
-    This is the place where common resources which can be used by multiple projects reside. The idea is not have to duplicate 
-    things such as `logback.xml`, `web.xml`, keystores and so on across the other modules. These resources are 
-    copied using the `maven-dependency-plugin`.
-    * [strongbox-common-resources]/  
-        Contains `logback.xml` and keystores.
-    * [strongbox-storage-resources]/  
-        * [strongbox-storage-api-resources]/  
-            Contains the `strongbox.xml` configuration file.
-        * [strongbox-web-resources]/  
-            Contains the `web.xml`and the Jetty configuration files.
-* [strongbox-rest-client]/  
+
+* [strongbox-commons] 
+    This is where the most common code that is shared across modules resides.
+
+* [strongbox-configuration]
```

Comment:
There is a missing link here. Also, would you mind making the text for all of the modules on a new line as they were? This is done by adding a 2 white spaces at the end:
 ```suggestion
* [strongbox-configuration]  
```